src/signer/canister/src/lib.rs

```diff
@@ -134,7 +134,7 @@ async fn generic_sign_with_ecdsa(
     payment: Option<PaymentType>,
     arg: SignWithEcdsaArgument,
 ) -> Result<(SignWithEcdsaResponse,), GenericSignWithEcdsaError> {
-    let fee = 1_000_000_000;
```

Comment:
Could we add a comment of where each price comes from? Just a comment that it comes from `scripts/check-pricing`.